docs/README.md

```diff
@@ -1,24 +1,46 @@
 # Compiling Membrane Curvature's Documentation
 
-The docs for this project are built with [Sphinx](http://www.sphinx-doc.org/en/master/).
-To compile the docs, first ensure that Sphinx and the ReadTheDocs theme are installed.
+The docs for this project are built with [Sphinx](http://www.sphinx-doc.org/en/master/) using the [mdanalysis-sphinx-theme](https://github.com/MDAnalysis/mdanalysis-sphinx-theme).
+To compile the docs, first ensure that:
 
+1. All necessary dependencies are installed.
+1. The editable version of this repo is installed.
```

Comment:
This isn't necessarily a requirement -- so long as the docs don't execute code that's changed, I think you could get by with a non-editable install :-)